tcbuilder/backend/platform.py

```
@@ -1881,6 +1881,23 @@ def push_fuse(credentials, target, version, fuse_file, hardwareids, *,
+    """Determine the number of fuses based on the secure-boot technology
+
+    :param tech: The secure-boot technology being used.
+    :returns: An int with the number of fuses
+    """
+
+    if tech == "hab":
+        fuse_num = 8
+    elif tech == "ahab":
+        fuse_num = 16
+    else:
+        assert False, f"Unhandled Secure Boot technology '{tech}'."
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)